### PR TITLE
Bug/lin kk tau spacing

### PR DESCRIPTION
**Pull request by BGerwe**
Changes to address #69. The actual changes were fairly simple, but reference that issue for a longer discussion about the impact to LinKK functionality.

### CODE REVIEW

File: impedance/validation.py [hunk 1]
```diff
@@ -80,8 +80,8 @@ def linKK(f, Z, c=0.85, max_M=50):
     def get_tc_distribution(f, M):
         """ Returns the distribution of time constants for the linKK method """
 
-        t_max = 1/np.min(f)
-        t_min = 1/np.max(f)
+        t_max = 1/(2 * np.pi * np.min(f))
```
Comment (mdmurbach) [inline impedance/validation.py]: The units here seem weird to me. I agree that it's what they have in their paper, and it definitely works/matches better. Just seems like the radians are disappearing? Or the units of tau aren't seconds? 😕

File: impedance/validation.py [hunk 2]
```diff
@@ -104,38 +103,39 @@ def get_tc_distribution(f, M):
...
 
     p_values = result['x']
-    mu = calc_mu(p_values[1:])
+    mu = calc_mu(p_values)
 
     return p_values, mu
 
 
-def eval_linKK(Rs, ts, f):
+def eval_linKK(Rs, R0, ts, f):
```
Comment (mdmurbach) [inline impedance/validation.py]: Great job breaking out R0 into a constant instead of a fit parameter 👑

File: impedance/validation.py (hunk 1 above)
Comment (BGerwe) [reply impedance/validation.py]: Yeah it's weird. I think it comes from the radians being assumed in angular frequencies and cycles (or waveforms) being assumed in linear frequency. Then 2 π is just the conversion of radians to cycles.

File: impedance/validation.py (hunk 2 above)
Comment (BGerwe) [reply impedance/validation.py]: Thanks! I looked at trying to use the approach for holding constants in circuit models, but this seemed more straight forward.

**Pull request merged**